internal/pkg/template/templates/task/cf.yml

```diff
@@ -57,14 +57,14 @@ Resources:
           Name: !Ref TaskName{{if .EnvVars}}
           Environment:{{range $name, $value := .EnvVars}}
           - Name: {{$name}}
-            Value: {{$value}}{{end}}{{end}}
+            Value: {{$value | printf "%q"}}{{end}}{{end}}
```

Comment:
Awesome this is much needed! For example without it, `{""key1"":""val1"",""key2"":5}` would get interpreted as a JSON object instead of a JSON string.

Reply:
yeah I ran into that during testing of the other portion of this change! haha